src/program-editors.rst

```diff
@@ -39,4 +39,252 @@ TODO: work on version control systems
 Accessibility
 =============
 
-TODO: accessibility-related discussion
\ No newline at end of file
+Introduction and Overview
+----------------------------
+
+Accessibility in computing refers to the development and design of both software and hardware systems to be usable by people with disabilities. Broadly, "Providing accessibility means removing barriers that prevent people with disabilities from participating in substantial life activities, including the use of services, products, and information." :cite:`bergman1995towards`. This includes providing screen readers for those with visual impairments and alternative input devices for those with motor impairments. 
```

Comment:
For long quotations, it would be helpful to use the block quotation format that sphinx supports